iam.tf

```diff
@@ -90,6 +90,33 @@ resource "aws_iam_policy_attachment" "network" {
   policy_arn = "${aws_iam_policy.network.arn}"
 }
 
+# Attach an additional policy for KMS
+
+data "aws_iam_policy_document" "kms" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "kms:Decrypt",
+    ]
+
+    resources = [
+      "${var.kms_key_arn == "" ? data.aws_kms_alias.default_lambda.target_key_id : var.kms_key_arn}",
```

Comment:
Does `target_key_id` return an ID or an ARN?

Terraform seems to have mixed up IDs and ARNs in their variable names and documentation so I don't trust anything related to them unless it's been seen in action. Have you tried doing a test? Any of the ones in the tests dir should do it.

Reply:
Fixed this, turns out there is an arn attribute I somehow missed on aws_kms_alias. I switched to using that.